### DISCUSSION TITLE
Bump eslint version

### DISCUSSION THREAD

**Discussion by mraballard** [vercel/next.js#68903] [Help] [answered]
### Summary

I'm trying to bump my eslint package to the latest version 9.9.0, however I'm running into dependency issues with next-eslint-config-next package which uses an older version. The current version in your package.json is 8.56.0, quite behind the current 9.9.0 version. Are there plans to upgrade this package anytime soon?

**Comment by Juneezee** [comment, marked as answer]
Next.js doesn't support ESLint 9 yet. There is an issue tracking this, see https://github.com/vercel/next.js/issues/64409

**Reply by mraballard** [reply]
Thank you @Juneezee
